ewms_pilot/pilot.py

```diff
@@ -246,6 +250,10 @@ async def process_msg_task(
     LOGGER.info("Sending return message...")
     await pub.send(out_msg)
 
+    # cleanup -- on success only
+    if not keep_debug_dir:
+        debug_subdir.unlink()  # rm
```

Comment:
I think this needs to be `shutil.rmtree()`.  `unlink` only works on files, not dirs, and deleting a dir needs to be empty with `rmdir`.

Reply:
good catch